desktop/package/README.md

```diff
@@ -42,7 +48,7 @@ Haveno data folder on Mac: `/Users/<username>/Library/Application Support/Haveno
     6. Click "OK" to save the changes and exit the dialog box.
     7. Windows will download and install the required files and components to enable the .NET Framework 3.5. This may take several minutes, depending on your internet connection speed and system configuration.
     8. Once the installation is complete, you will need to restart your computer to apply the changes.
-2. Install Wix Toolset 3: https://github.com/wixtoolset/wix3/releases/tag/wix314rtm
+2. Install Wix Toolset 3: <https://github.com/wixtoolset/wix3/releases/tag/wix314rtm>
```

Comment:
Why add `<>` here?

Reply:
I think that was my automatic linter. [MD034](https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md) says that some MarkDown parsers don't understand bare URLs and that angle brackets are the "proper" syntax to use.